src/core/utils/modal.tsx

```diff
@@ -19,6 +19,7 @@ type ModalProps = {
   screenReaderModalDescriptionText: string;
   classNames?: string;
   dataCy?: string;
+  orientation?: "right";
```

Comment:
Please reconsider this structure.

Instead of making the prop an optional literal type with a single value I suggest using a non-optional literal with multiple values e.g. "center" | "right" and provide a default value of "center".

This way you also avoid the current detection where any `orientation` value will add the `modal-btn-close--offset` class. I would have expected that only the `right` value would need the `modal-btn-close--offset` class.

Reply:
I remade it a bit differently - to a boolean. We currently only have two types of modals - centered with offset and sliders that come from the right side. I overcomplicated it with the last implementation.